certbot/certbot/util.py

```diff
@@ -116,7 +117,7 @@ def run_script(params, log=logger.error):
     return proc.stdout, proc.stderr
 
 
-def exe_exists(exe):
+def exe_exists(exe: Optional[str]) -> bool:
```

Comment:
Why `Optional`? `mypy` seems happy without it, so far.

Reply:
This is the remaining of a previous fix. Now the nullity of the exe path is tested upfront, and so the check here is not needed anymore.